### DISCUSSION TITLE
Datastore query type error dish.restaurantID not a function

### DISCUSSION THREAD

**Discussion by Chloedevelopers** [community/community#46592] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

`useeffect(() => {
Datastore.query(Dish,(dish) => dish.restaurantID("eq" , I'd)).then(setDishes);
}, []);`
I get this error : dish.restaurantID is not a function please help me out

**Comment by jge162** [comment]
try this instead ->

`Datastore.query(Dish, (dish) => dish.restaurantID === 'I'd').then(setDishes);`

**Comment by Chloedevelopers** [comment]
I get an error

**Reply by jge162** [reply]
I cannot help with "I get an error" need more details please. thanks

**Comment by Chloedevelopers** [comment]
I'm trying to query dish from datastore but I get an error type
dish.restaurantID not a function. I'm following up your UberEATS tutorial
to build UberEATS user app when I come across this issue I'm using  expo
and I have all dependencies. Let me send the code.

**Comment by Chloedevelopers** [comment]
ubereats user day 2
screens
restaurantDetailScreens
index.js

**Comment by Chloedevelopers** [comment]
I do I have to go about it to create the restaurantID as function please
help me out because I'm following the asset bundle provided

**Comment by Chloedevelopers** [comment]
Can you try to look at the source code
UberEATS user day 2
src
Screens
RestaurantDetailsScreen
Please if there is any change you can provide me with the source code

**Comment by Chloedevelopers** [comment]
Thanks I have solved the problem

**Reply by jge162** [reply]
Sounds good.
